10/umbraco-cms/fundamentals/setup/upgrading/version-specific/minor-upgrades-for-umbraco-8.md

```diff
@@ -0,0 +1,148 @@
+---
+description: This articles provides details on how to upgrade to the next minor version when using Umbraco 8.
+---
+
+# Minor upgrades for Umbraco 8
+
+Sometimes there are exceptions to these guidelines, which are listed in the **[version-specific guide](README.md)**.
+
+## Note
+
+It is necessary to run the upgrade installer on each environment of your Umbraco site. So if you want to update your staging and your live site then you need to repeat the steps below and make sure that you click through the install screens so that your upgrade is complete.
```

Comment:
```suggestion
It is necessary to run the upgrade installer on each environment of your Umbraco site. If you want to update your staging and live site you need to repeat the steps below. Make sure you click through the install screens so that your upgrade is complete.
```